.github/workflows/ci.yaml

```diff
@@ -1,9 +1,6 @@
 name: CI
 
 env:
-  # A suitable URL for the test database.
-  DATABASE_URL: postgres://postgres:postgres@127.0.0.1:5432/river_dev?sslmode=disable
```

Comment:
This wasn't needed anywhere so I took it out so that the env is cleaner for tests where we want to work with it (i.e. we don't have to unset `DATABASE_URL` before testing something).